modules/python3qt/pythoneditorwidget.cpp

```diff
@@ -225,6 +232,7 @@ void PythonEditorWidget::closeEvent(QCloseEvent* event) {
     settings.beginGroup(objectName());
     settings.setValue("appendLog", appendLog_->isChecked());
     settings.setValue("lastScript", scriptFileName_.c_str());
+    settings.setValue("source", pythonCode_->toPlainText());
```

Comment:
will this be the code edited last in the editor?

Reply:
This will save the source that currently are in the editor when the editor closes. 

Maybe we should also store the code upon run (prior to run). My though with the change is to recover code lost after either a crash or when you forget to save the code.

Reply:
then prior run would be a good idea as well.
But: what happens if you editing a file, edit the script in Inviwo, run it (it will be stored), and then it crashes.
Upon a restart, the file will be loaded anyway.